<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/logos/icons/logo.svg</normaloff>:/logos/icons/logo.svg</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionWavefront_obj"/>
    </widget>
    <addaction name="actionNew_scene"/>
    <addaction name="action_Open_scene"/>
    <addaction name="actionSave_scene"/>
    <addaction name="actionSave_scene_as"/>
    <addaction name="separator"/>
    <addaction name="menuImport"/>
    <addaction name="separator"/>
    <addaction name="actionSave_screenshot"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="m_qMenuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="m_qMenuView">
    <property name="accessibleName">
     <string extracomment="The View menu">m_qMenuView</string>
    </property>
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit_2">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="m_qToolbarsMenu">
    <property name="title">
     <string>Toolbars</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit_2"/>
   <addaction name="m_qMenuTools"/>
   <addaction name="m_qMenuView"/>
   <addaction name="m_qToolbarsMenu"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_scene"/>
   <addaction name="action_Open_scene"/>
   <addaction name="actionSave_scene"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open_scene">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/file/icons/file_icon.png</normaloff>
     <normalon>:/file/icons/file_icon.png</normalon>:/file/icons/file_icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open scene...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionSave_scene">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/file/icons/diskette_icon.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save scene</string>
   </property>
  </action>
  <action name="actionSave_scene_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save scene as...</string>
   </property>
  </action>
  <action name="actionWavefront_obj">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Wavefront obj...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normalon>:/file/icons/door_icon.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Exit.</string>
   </property>
  </action>
  <action name="actionNew_scene">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/file/icons/document_icon.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New scene</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionSave_screenshot">
   <property name="text">
    <string>Save screenshot...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
